ams/server/src/main/resources/mysql/upgrade-0.5.0-to-0.5.1.sql

```diff
@@ -1,2 +1,4 @@
 UPDATE `table_optimizing_process` SET `status`  = 'CLOSED' WHERE `status` ='RUNNING';
 UPDATE `table_runtime` SET `optimizing_status`  = 'IDLE';
+
+ALTER TABLE `table_runtime` ADD `format` varchar(32)  NOT NULL COMMENT "format";
```

Comment:
Will this feature be include into `0.5.1` ?

Reply:
In my consider once the pull request is merged, there should be an SQL for this issue.

Reply:
I think you could add this to  upgrade-0.5.1-to-0.6.0.sql.

Or we keep a file of ugprade-to-next.sql in master branch ? @zhoujinsong

Reply:
Yes, we may keep a `uprade.sql` scripts to store the upgrade SQL in master branch as we do not know the exact version the feature will be published as we merge the PR.